product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx

```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```

Comment:
In CentOS7 command is `yum` not `dnf`

Reply:
dnf ships with Centos7, it just isn't default. Yum is also present. Centos8 sees Dnf as default (and for all, going forward).